wordpress-mobile/WordPress-iOS
# Stats Revamp: Fix Followers Details Screen Followers Total

This PR adds more stability for the top Followers Total card on the 2nd level Stats Insights screen.  During testing it was noted that Followers Total could be showing 0 on the top card even though there are followers.

I hadn't appreciated that StatsInsightsStore has 2 methods of requesting followers for example:
 * dotComFollowers
 * allDotComFollowers

For the purposes of the Followers Total card summary, I've changed the call to use the dotComFollowers which returns quicker and is consistent with the Top Level Followers Total insights card

Ref #18605

Screenshot below of the problem.  Note that this inconsistent Followers Total count does not always manifest (I could only get it to happen 1 out of 20 times)

<img width="333" alt="image" src="https://user-images.githubusercontent.com/88816724/176160493-74bc9b61-989e-4566-a8fb-bd154b4b7849.png">

Expected result:
<img width="333" alt="image" src="https://user-images.githubusercontent.com/88816724/176160597-3352b566-63bf-4767-990e-a9cf522aaaa2.png">


To test:
	1. enable both of the new stats feature flags statsNewAppearance and statsNewInsights via AppConfiguration.showsStatsRevampV2
	2. From top level Stats Insights screen, Find the Followers Total card (or add if needed) and take note of the count.  Then tap "View More" on the Followers Total card. You should be taken to the 2nd Level Insights Details and should see Followers Total card at the top
	3. The count of the Followers Total card on the 2nd level should match the count of the Followers Total card on the top level
Switch sites and continue to test this for other sites


## Regression Notes
1. Potential unintended areas of impact
Stats revamp v2

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manually tested

3. What automated tests I added (or what prevented me from doing so)
None

PR submission checklist:
